userspace/libsinsp/container_info.h

```diff
@@ -36,14 +36,16 @@ enum sinsp_container_type
 	CT_CUSTOM = 5,
 	CT_CRI = 6,
 	CT_CONTAINERD = 7,
+	CT_CRIO = 8,
 };
 
 // Docker and CRI-compatible runtimes are very similar
 static inline bool is_docker_compatible(sinsp_container_type t)
 {
 	return t == CT_DOCKER ||
 		t == CT_CRI ||
-		t == CT_CONTAINERD;
+		t == CT_CONTAINERD ||
+		t == CT_CRIO;
```

Comment:
fine for now, but if we have more of these things, it may start to become useful to define classes for these things that implement stuff like

-is_docker_compatible
-get_cgroup_layout